src/components/grid-selector.less

```diff
@@ -0,0 +1,41 @@
+@grid-selector-icon-size: 32px;
+@grid-selector-item-width: 3 * @grid-selector-icon-size;
+
+.GridSelector {
+  margin: 0;
+  padding: 0;
+  display: grid;
+  gap: @edu-button-spacing;
+  grid-template-columns: repeat(auto-fill, minmax(min(@grid-selector-item-width, 100%), 1fr));
+}
+
+.GridSelector-item {
+  display: grid;
+  grid-auto-flow: row;
+  align-items: center;
+  justify-items: center;
+  gap: 2px;
+  height: auto;
+  width: 100%;
+  margin: 0;
+  padding: @edu-button-spacing;
+  border: 1px solid transparent;
+  border-radius: @edu-border-radius-base;
+  cursor: pointer;
+  user-select: none;
+  outline: none !important; // Removes user agent black border when focused by keyboard
```

Comment:
Should this maybe go into `&.GridSelector-item--selected &:focus` ? Or maybe in it's own mixin?

Reply:
It should and it did (both of your suggestions) :smile: